From a704d0b6cfea091274bd79aca6d15f19b4f6e5b5 Mon Sep 17 00:00:00 2001
From: Marinus Schraal <mschraal@gnome.org>
Date: Sat, 6 Jan 2018 15:04:10 +0100
Subject: [PATCH] meson: Introspection fix

The meson build did not add the extractdummy.c to the sources, which
contains introspection annotations.

https://bugzilla.gnome.org/show_bug.cgi?id=792272
---
 libmediaart/meson.build | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/libmediaart/meson.build b/libmediaart/meson.build
index c02aee0..265a9ab 100644
--- a/libmediaart/meson.build
+++ b/libmediaart/meson.build
@@ -6,6 +6,10 @@ libmediaart_public_headers = [
   'mediaart.h',
 ]
 
+libmediaart_introspection_sources = [
+  'extractdummy.c'
+]
+
 libmediaart_sources = [
   'cache.c',
   'extract.c',
@@ -38,7 +42,7 @@ libmediaart = shared_library(
 )
 
 libmediaart_gir_and_typelib = gnome.generate_gir(libmediaart,
-  sources: libmediaart_sources + libmediaart_public_headers,
+  sources: libmediaart_sources + libmediaart_introspection_sources + libmediaart_public_headers,
   nsversion: libmediaart_api_version,
   namespace: 'MediaArt',
   identifier_prefix: 'MediaArt',
